runtime/altair/src/migrations.rs

```diff
@@ -64,7 +66,7 @@ mod asset_registry {
 
 	impl OnRuntimeUpgrade for CrossChainTransferabilityMigration {
 		fn on_runtime_upgrade() -> Weight {
-			if VERSION.spec_version > 1028 {
+			if VERSION.spec_version > 1030 {
```

Comment:
This will run on `Algol` then? 

@NunoAlexandre IIRC you already tested that another run does not harm, right?

Reply:
I have an idea 👍

Reply:
@mustermeiszer https://github.com/centrifuge/centrifuge-chain/pull/1510 👀

Reply:
Nice!